visitz/Visitz/ViewModels/NotesViewModel.cs

```diff
@@ -44,9 +47,21 @@ public async Task CaseDetailsTapped()
             await CaseloadItemDetailsPage.Open(VisitzPage, CaseIncident.CaseIncidentNumber);
         }
 
+        [RelayCommand]
+        public void RefreshNotes()
+        {
+            if (CaseIncident.CaseIncidentNumber == null
+                || CaseIncident.EntityType == null)
+            {
+                IsRefreshing = false;
+                return;
+            }
+            WeakReferenceMessenger.Default.Send(GetNotesService.MakeStartMessage(CaseIncident.CaseIncidentNumber, CaseIncident.EntityType));
```

Comment:
Line exceeds 120 characters, please break it up. Something like:

```suggestion
var entityTuple = (CaseIncident.CaseIncidentNumber, CaseIncident.EntityType);
WeakReferenceMessenger.Default.Send(GetNotesService.MakeStartMessage(entityTuple));
```